.github/workflows/release-test.yml

```diff
@@ -28,7 +31,7 @@ jobs:
 
       - name: Publish with Verdaccio
         run: |
-          git checkout ${{ github.head_ref }}
+          git switch -c "pull-request"
```

Comment:
`action-checkout` left git in "detached state". Original 
`git checkout ${{ github.head_ref }}`

Was aimed to fix this and make state attached again. But this notation doesn't work with pull request from forks (as this one) because `github.head_ref` is pointing to the ref local to the fork not an original repo. 

`git switch -c "pull-request"` also do the trick, and works with pull request as well.